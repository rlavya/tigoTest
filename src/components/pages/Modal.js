import React, { Component } from "react";
import RaisedButton from "material-ui/RaisedButton";
import FlatButton from "material-ui/FlatButton";
import TextField from "material-ui/TextField";
import PageTitle from "../components/PageTitle";
import { lightBlue200, indigo300 } from "material-ui/styles/colors";

import Dialog from "material-ui/Dialog";

class Modal extends Component {
  state = { open: true };
  handleOpen = () => {
    this.setState({ open: true });
  };

  handleClose = () => {
    this.setState({ open: false });
  };
  render() {
    return (
      <PageTitle page="Login">
        <div className="max-width-4 mx-auto">
          <div className="clearfix center">
            <h1 className="regular text-primary h3">
              Ingresa tus credenciales
            </h1>

            <TextField
              underlineFocusStyle={{ borderColor: lightBlue200 }}
              floatingLabelStyle={{ color: indigo300 }}
              floatingLabelText="DMS ID"
              type="tel"
              pattern="[0-9]*"
            />
            <br />
            <TextField
              underlineFocusStyle={{ borderColor: lightBlue200 }}
              floatingLabelStyle={{ color: indigo300 }}
              floatingLabelText="ePIN"
              type="tel"
              pattern="[0-9]*"
            />
            <br />

            <RaisedButton
              className="mt2"
              secondary={true}
              label="Iniciar"
              onClick={this.handleOpen}
            />
          </div>
          <Dialog
            modal={false}
            open={this.state.open}
            onRequestClose={this.handleClose}
            className="center"
          >
            <h2 className="light text-primary">Ingresa tu PIN:</h2>
            <TextField
              underlineFocusStyle={{ borderColor: lightBlue200 }}
              floatingLabelStyle={{ color: indigo300 }}
              floatingLabelText="PIN"
              type="tel"
              pattern="[0-9]*"
            />
            <div className="mt2">
              <RaisedButton
                label="Continuar"
                secondary={true}
                onClick={this.handleClose}
              />
              <br />
              <FlatButton
                label="Cancelar"
                secondary={true}
                keyboardFocused={true}
                onClick={this.handleClose}
              />
            </div>
          </Dialog>
        </div>
      </PageTitle>
    );
  }
}

export default Modal;
