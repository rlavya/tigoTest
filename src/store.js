import { applyMiddleware, createStore } from "redux";
import { createLogger } from "redux-logger";
import { composeWithDevTools } from "redux-devtools-extension/developmentOnly";
import thunk from "redux-thunk";
import reducer from "./reducer";

const getMiddleware = () => {
  return applyMiddleware(createLogger(), thunk);
};

const store = createStore(reducer, composeWithDevTools(getMiddleware()));

export default store;
