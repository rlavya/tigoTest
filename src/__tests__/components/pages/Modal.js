import Modal from "../../../components/pages/Modal";
import renderer from "react-test-renderer";
import Enzyme, { shallow, render, mount } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16';
import React from "react";
import store from "../../../store";
import { Provider } from "react-redux";

Enzyme.configure({ adapter: new Adapter() });

describe('Modal component', () => {

  it("renders correctly", () => {
    const component = shallow(
      <Modal />
    );

    expect(component).toMatchSnapshot();
  });

});
