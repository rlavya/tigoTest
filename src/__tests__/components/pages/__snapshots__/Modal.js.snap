// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal component renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Modal />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Connect(PageTitle)
          page="Login"
/>,
        <div
          className="clearfix center"
>
          <h1
                    className="regular text-primary h3"
          >
                    Ingresa tus credenciales
          </h1>
          <TextField
                    disabled={false}
                    floatingLabelFixed={false}
                    floatingLabelStyle={
                              Object {
                                        "color": "#7986cb",
                                      }
                    }
                    floatingLabelText="DMSID"
                    fullWidth={false}
                    multiLine={false}
                    rows={1}
                    type="number"
                    underlineFocusStyle={
                              Object {
                                        "borderColor": "#81d4fa",
                                      }
                    }
                    underlineShow={true}
          />
          <br />
          <TextField
                    disabled={false}
                    floatingLabelFixed={false}
                    floatingLabelStyle={
                              Object {
                                        "color": "#7986cb",
                                      }
                    }
                    floatingLabelText="ePIN"
                    fullWidth={false}
                    multiLine={false}
                    rows={1}
                    type="number"
                    underlineFocusStyle={
                              Object {
                                        "borderColor": "#81d4fa",
                                      }
                    }
                    underlineShow={true}
          />
          <br />
          <RaisedButton
                    className="mt2"
                    disabled={false}
                    fullWidth={false}
                    label="Iniciar"
                    labelPosition="after"
                    onClick={[Function]}
                    primary={false}
                    secondary={true}
          />
</div>,
        <Dialog
          autoDetectWindowHeight={true}
          autoScrollBodyContent={false}
          className="center"
          modal={false}
          onRequestClose={[Function]}
          open={true}
          repositionOnUpdate={true}
>
          <h2
                    className="light text-primary"
          >
                    Ingresa tu clave:
          </h2>
          <TextField
                    disabled={false}
                    floatingLabelFixed={false}
                    floatingLabelStyle={
                              Object {
                                        "color": "#7986cb",
                                      }
                    }
                    floatingLabelText="CLAVE"
                    fullWidth={false}
                    multiLine={false}
                    rows={1}
                    type="number"
                    underlineFocusStyle={
                              Object {
                                        "borderColor": "#81d4fa",
                                      }
                    }
                    underlineShow={true}
          />
          <div
                    className="mt2"
          >
                    <RaisedButton
                              disabled={false}
                              fullWidth={false}
                              label="Continuar"
                              labelPosition="after"
                              onClick={[Function]}
                              primary={false}
                              secondary={true}
                    />
                    <br />
                    <FlatButton
                              disabled={false}
                              fullWidth={false}
                              keyboardFocused={true}
                              label="Cancelar"
                              labelPosition="after"
                              labelStyle={Object {}}
                              onClick={[Function]}
                              onKeyboardFocus={[Function]}
                              onMouseEnter={[Function]}
                              onMouseLeave={[Function]}
                              onTouchStart={[Function]}
                              primary={false}
                              secondary={true}
                    />
          </div>
</Dialog>,
      ],
      "className": "max-width-4 mx-auto",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "page": "Login",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h1
              className="regular text-primary h3"
>
              Ingresa tus credenciales
</h1>,
            <TextField
              disabled={false}
              floatingLabelFixed={false}
              floatingLabelStyle={
                            Object {
                                          "color": "#7986cb",
                                        }
              }
              floatingLabelText="DMSID"
              fullWidth={false}
              multiLine={false}
              rows={1}
              type="number"
              underlineFocusStyle={
                            Object {
                                          "borderColor": "#81d4fa",
                                        }
              }
              underlineShow={true}
/>,
            <br />,
            <TextField
              disabled={false}
              floatingLabelFixed={false}
              floatingLabelStyle={
                            Object {
                                          "color": "#7986cb",
                                        }
              }
              floatingLabelText="ePIN"
              fullWidth={false}
              multiLine={false}
              rows={1}
              type="number"
              underlineFocusStyle={
                            Object {
                                          "borderColor": "#81d4fa",
                                        }
              }
              underlineShow={true}
/>,
            <br />,
            <RaisedButton
              className="mt2"
              disabled={false}
              fullWidth={false}
              label="Iniciar"
              labelPosition="after"
              onClick={[Function]}
              primary={false}
              secondary={true}
/>,
          ],
          "className": "clearfix center",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Ingresa tus credenciales",
              "className": "regular text-primary h3",
            },
            "ref": null,
            "rendered": "Ingresa tus credenciales",
            "type": "h1",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "disabled": false,
              "floatingLabelFixed": false,
              "floatingLabelStyle": Object {
                "color": "#7986cb",
              },
              "floatingLabelText": "DMSID",
              "fullWidth": false,
              "multiLine": false,
              "rows": 1,
              "type": "number",
              "underlineFocusStyle": Object {
                "borderColor": "#81d4fa",
              },
              "underlineShow": true,
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": "br",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "disabled": false,
              "floatingLabelFixed": false,
              "floatingLabelStyle": Object {
                "color": "#7986cb",
              },
              "floatingLabelText": "ePIN",
              "fullWidth": false,
              "multiLine": false,
              "rows": 1,
              "type": "number",
              "underlineFocusStyle": Object {
                "borderColor": "#81d4fa",
              },
              "underlineShow": true,
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": "br",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "className": "mt2",
              "disabled": false,
              "fullWidth": false,
              "label": "Iniciar",
              "labelPosition": "after",
              "onClick": [Function],
              "primary": false,
              "secondary": true,
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "autoDetectWindowHeight": true,
          "autoScrollBodyContent": false,
          "children": Array [
            <h2
              className="light text-primary"
>
              Ingresa tu clave:
</h2>,
            <TextField
              disabled={false}
              floatingLabelFixed={false}
              floatingLabelStyle={
                            Object {
                                          "color": "#7986cb",
                                        }
              }
              floatingLabelText="CLAVE"
              fullWidth={false}
              multiLine={false}
              rows={1}
              type="number"
              underlineFocusStyle={
                            Object {
                                          "borderColor": "#81d4fa",
                                        }
              }
              underlineShow={true}
/>,
            <div
              className="mt2"
>
              <RaisedButton
                            disabled={false}
                            fullWidth={false}
                            label="Continuar"
                            labelPosition="after"
                            onClick={[Function]}
                            primary={false}
                            secondary={true}
              />
              <br />
              <FlatButton
                            disabled={false}
                            fullWidth={false}
                            keyboardFocused={true}
                            label="Cancelar"
                            labelPosition="after"
                            labelStyle={Object {}}
                            onClick={[Function]}
                            onKeyboardFocus={[Function]}
                            onMouseEnter={[Function]}
                            onMouseLeave={[Function]}
                            onTouchStart={[Function]}
                            primary={false}
                            secondary={true}
              />
</div>,
          ],
          "className": "center",
          "modal": false,
          "onRequestClose": [Function],
          "open": true,
          "repositionOnUpdate": true,
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Ingresa tu clave:",
              "className": "light text-primary",
            },
            "ref": null,
            "rendered": "Ingresa tu clave:",
            "type": "h2",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "disabled": false,
              "floatingLabelFixed": false,
              "floatingLabelStyle": Object {
                "color": "#7986cb",
              },
              "floatingLabelText": "CLAVE",
              "fullWidth": false,
              "multiLine": false,
              "rows": 1,
              "type": "number",
              "underlineFocusStyle": Object {
                "borderColor": "#81d4fa",
              },
              "underlineShow": true,
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <RaisedButton
                  disabled={false}
                  fullWidth={false}
                  label="Continuar"
                  labelPosition="after"
                  onClick={[Function]}
                  primary={false}
                  secondary={true}
/>,
                <br />,
                <FlatButton
                  disabled={false}
                  fullWidth={false}
                  keyboardFocused={true}
                  label="Cancelar"
                  labelPosition="after"
                  labelStyle={Object {}}
                  onClick={[Function]}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onTouchStart={[Function]}
                  primary={false}
                  secondary={true}
/>,
              ],
              "className": "mt2",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "disabled": false,
                  "fullWidth": false,
                  "label": "Continuar",
                  "labelPosition": "after",
                  "onClick": [Function],
                  "primary": false,
                  "secondary": true,
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {},
                "ref": null,
                "rendered": null,
                "type": "br",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "disabled": false,
                  "fullWidth": false,
                  "keyboardFocused": true,
                  "label": "Cancelar",
                  "labelPosition": "after",
                  "labelStyle": Object {},
                  "onClick": [Function],
                  "onKeyboardFocus": [Function],
                  "onMouseEnter": [Function],
                  "onMouseLeave": [Function],
                  "onTouchStart": [Function],
                  "primary": false,
                  "secondary": true,
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
            ],
            "type": "div",
          },
        ],
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Connect(PageTitle)
            page="Login"
/>,
          <div
            className="clearfix center"
>
            <h1
                        className="regular text-primary h3"
            >
                        Ingresa tus credenciales
            </h1>
            <TextField
                        disabled={false}
                        floatingLabelFixed={false}
                        floatingLabelStyle={
                                    Object {
                                                "color": "#7986cb",
                                              }
                        }
                        floatingLabelText="DMSID"
                        fullWidth={false}
                        multiLine={false}
                        rows={1}
                        type="number"
                        underlineFocusStyle={
                                    Object {
                                                "borderColor": "#81d4fa",
                                              }
                        }
                        underlineShow={true}
            />
            <br />
            <TextField
                        disabled={false}
                        floatingLabelFixed={false}
                        floatingLabelStyle={
                                    Object {
                                                "color": "#7986cb",
                                              }
                        }
                        floatingLabelText="ePIN"
                        fullWidth={false}
                        multiLine={false}
                        rows={1}
                        type="number"
                        underlineFocusStyle={
                                    Object {
                                                "borderColor": "#81d4fa",
                                              }
                        }
                        underlineShow={true}
            />
            <br />
            <RaisedButton
                        className="mt2"
                        disabled={false}
                        fullWidth={false}
                        label="Iniciar"
                        labelPosition="after"
                        onClick={[Function]}
                        primary={false}
                        secondary={true}
            />
</div>,
          <Dialog
            autoDetectWindowHeight={true}
            autoScrollBodyContent={false}
            className="center"
            modal={false}
            onRequestClose={[Function]}
            open={true}
            repositionOnUpdate={true}
>
            <h2
                        className="light text-primary"
            >
                        Ingresa tu clave:
            </h2>
            <TextField
                        disabled={false}
                        floatingLabelFixed={false}
                        floatingLabelStyle={
                                    Object {
                                                "color": "#7986cb",
                                              }
                        }
                        floatingLabelText="CLAVE"
                        fullWidth={false}
                        multiLine={false}
                        rows={1}
                        type="number"
                        underlineFocusStyle={
                                    Object {
                                                "borderColor": "#81d4fa",
                                              }
                        }
                        underlineShow={true}
            />
            <div
                        className="mt2"
            >
                        <RaisedButton
                                    disabled={false}
                                    fullWidth={false}
                                    label="Continuar"
                                    labelPosition="after"
                                    onClick={[Function]}
                                    primary={false}
                                    secondary={true}
                        />
                        <br />
                        <FlatButton
                                    disabled={false}
                                    fullWidth={false}
                                    keyboardFocused={true}
                                    label="Cancelar"
                                    labelPosition="after"
                                    labelStyle={Object {}}
                                    onClick={[Function]}
                                    onKeyboardFocus={[Function]}
                                    onMouseEnter={[Function]}
                                    onMouseLeave={[Function]}
                                    onTouchStart={[Function]}
                                    primary={false}
                                    secondary={true}
                        />
            </div>
</Dialog>,
        ],
        "className": "max-width-4 mx-auto",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "page": "Login",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h1
                className="regular text-primary h3"
>
                Ingresa tus credenciales
</h1>,
              <TextField
                disabled={false}
                floatingLabelFixed={false}
                floatingLabelStyle={
                                Object {
                                                "color": "#7986cb",
                                              }
                }
                floatingLabelText="DMSID"
                fullWidth={false}
                multiLine={false}
                rows={1}
                type="number"
                underlineFocusStyle={
                                Object {
                                                "borderColor": "#81d4fa",
                                              }
                }
                underlineShow={true}
/>,
              <br />,
              <TextField
                disabled={false}
                floatingLabelFixed={false}
                floatingLabelStyle={
                                Object {
                                                "color": "#7986cb",
                                              }
                }
                floatingLabelText="ePIN"
                fullWidth={false}
                multiLine={false}
                rows={1}
                type="number"
                underlineFocusStyle={
                                Object {
                                                "borderColor": "#81d4fa",
                                              }
                }
                underlineShow={true}
/>,
              <br />,
              <RaisedButton
                className="mt2"
                disabled={false}
                fullWidth={false}
                label="Iniciar"
                labelPosition="after"
                onClick={[Function]}
                primary={false}
                secondary={true}
/>,
            ],
            "className": "clearfix center",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Ingresa tus credenciales",
                "className": "regular text-primary h3",
              },
              "ref": null,
              "rendered": "Ingresa tus credenciales",
              "type": "h1",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "disabled": false,
                "floatingLabelFixed": false,
                "floatingLabelStyle": Object {
                  "color": "#7986cb",
                },
                "floatingLabelText": "DMSID",
                "fullWidth": false,
                "multiLine": false,
                "rows": 1,
                "type": "number",
                "underlineFocusStyle": Object {
                  "borderColor": "#81d4fa",
                },
                "underlineShow": true,
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "br",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "disabled": false,
                "floatingLabelFixed": false,
                "floatingLabelStyle": Object {
                  "color": "#7986cb",
                },
                "floatingLabelText": "ePIN",
                "fullWidth": false,
                "multiLine": false,
                "rows": 1,
                "type": "number",
                "underlineFocusStyle": Object {
                  "borderColor": "#81d4fa",
                },
                "underlineShow": true,
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "br",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "className": "mt2",
                "disabled": false,
                "fullWidth": false,
                "label": "Iniciar",
                "labelPosition": "after",
                "onClick": [Function],
                "primary": false,
                "secondary": true,
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "autoDetectWindowHeight": true,
            "autoScrollBodyContent": false,
            "children": Array [
              <h2
                className="light text-primary"
>
                Ingresa tu clave:
</h2>,
              <TextField
                disabled={false}
                floatingLabelFixed={false}
                floatingLabelStyle={
                                Object {
                                                "color": "#7986cb",
                                              }
                }
                floatingLabelText="CLAVE"
                fullWidth={false}
                multiLine={false}
                rows={1}
                type="number"
                underlineFocusStyle={
                                Object {
                                                "borderColor": "#81d4fa",
                                              }
                }
                underlineShow={true}
/>,
              <div
                className="mt2"
>
                <RaisedButton
                                disabled={false}
                                fullWidth={false}
                                label="Continuar"
                                labelPosition="after"
                                onClick={[Function]}
                                primary={false}
                                secondary={true}
                />
                <br />
                <FlatButton
                                disabled={false}
                                fullWidth={false}
                                keyboardFocused={true}
                                label="Cancelar"
                                labelPosition="after"
                                labelStyle={Object {}}
                                onClick={[Function]}
                                onKeyboardFocus={[Function]}
                                onMouseEnter={[Function]}
                                onMouseLeave={[Function]}
                                onTouchStart={[Function]}
                                primary={false}
                                secondary={true}
                />
</div>,
            ],
            "className": "center",
            "modal": false,
            "onRequestClose": [Function],
            "open": true,
            "repositionOnUpdate": true,
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Ingresa tu clave:",
                "className": "light text-primary",
              },
              "ref": null,
              "rendered": "Ingresa tu clave:",
              "type": "h2",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "disabled": false,
                "floatingLabelFixed": false,
                "floatingLabelStyle": Object {
                  "color": "#7986cb",
                },
                "floatingLabelText": "CLAVE",
                "fullWidth": false,
                "multiLine": false,
                "rows": 1,
                "type": "number",
                "underlineFocusStyle": Object {
                  "borderColor": "#81d4fa",
                },
                "underlineShow": true,
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <RaisedButton
                    disabled={false}
                    fullWidth={false}
                    label="Continuar"
                    labelPosition="after"
                    onClick={[Function]}
                    primary={false}
                    secondary={true}
/>,
                  <br />,
                  <FlatButton
                    disabled={false}
                    fullWidth={false}
                    keyboardFocused={true}
                    label="Cancelar"
                    labelPosition="after"
                    labelStyle={Object {}}
                    onClick={[Function]}
                    onKeyboardFocus={[Function]}
                    onMouseEnter={[Function]}
                    onMouseLeave={[Function]}
                    onTouchStart={[Function]}
                    primary={false}
                    secondary={true}
/>,
                ],
                "className": "mt2",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "disabled": false,
                    "fullWidth": false,
                    "label": "Continuar",
                    "labelPosition": "after",
                    "onClick": [Function],
                    "primary": false,
                    "secondary": true,
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {},
                  "ref": null,
                  "rendered": null,
                  "type": "br",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "disabled": false,
                    "fullWidth": false,
                    "keyboardFocused": true,
                    "label": "Cancelar",
                    "labelPosition": "after",
                    "labelStyle": Object {},
                    "onClick": [Function],
                    "onKeyboardFocus": [Function],
                    "onMouseEnter": [Function],
                    "onMouseLeave": [Function],
                    "onTouchStart": [Function],
                    "primary": false,
                    "secondary": true,
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
              ],
              "type": "div",
            },
          ],
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
